### PR TITLE
Windows Component Service basic module and state

### PR DESCRIPTION
**Pull request by JulioECastro**
Using PowerShell and COM+ Object model I create this module and state to manage COM+ components. I hope it will be useful to anyone.
Regards.-

### CODE REVIEW

File: modules/win_complus.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+'''
+Microsoft Component Service management via powershell module
+
+:platform:      Windows
+
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
```
Comment (damon-atkins) [inline modules/win_complus.py]: from __future__ import unicode_literals

Please add the above to support different languages

**Comment by damon-atkins** [comment]
Please use utils/powershell.py (for Unix and Windows) if it needs to be enhanced i.e. a function that can be shared for new modules, please add it.

**Comment by damon-atkins** [comment]
Also use cmd.shell_info see other modules for example.

**Comment by damon-atkins** [comment]
Maybe stick to the same function names as pkg ?

File: modules/win_complus.py (hunk 1 above)
Comment (JulioECastro) [inline modules/win_complus.py]: done!

**Comment by JulioECastro** [comment]
Hi @damon-atkins. thanks for your commnets. I'll do the changes you suggest.
imports and shell_info:  ok.

You recommend use utils/powersheel . is it for checking existing powershell module? I don't need any ps module here. Could you send me a code example?

I see pkg functions... install and remove... but COM+ doesn't have the complexity of pkg management.

For future versions:
- Add component installations. adding dlls to com+ applications.
- Prebuild applications.
- Export (proxies and server applications)

**Comment by damon-atkins** [comment]
utils/powershell.py  more is about see if a module is available at this stage.  Plan is to add mot utils/functions which can be shared across unix/windows.    I developed the cmd.shell_info for powershell firstly as it took far to long to start to see if it was installed and what version it was (times the the number of modules using powershell) And thought I collect other shell info as well.

**Pull request merged**